indicators/NO_WECO_001-012/R/NO_WECO_001-012.qmd

```diff
@@ -0,0 +1,937 @@
+---
```

Comment:
I will perform my code review in a branch of the original repo: https://github.com/NINAnor/ecRxiv_WLEV
and just make the odd comment here for transparrency